crates/transaction-emitter-lib/src/args.rs

```diff
@@ -218,6 +218,18 @@ pub struct EmitArgs {
 
     #[clap(long)]
     pub coins_per_account_override: Option<u64>,
+
+    #[clap(long, value_parser = ConfigKey::<Ed25519PrivateKey>::from_encoded_string, requires = "proof_file_path", requires = "keyless_jwt")]
+    pub keyless_ephem_secret_key: Option<ConfigKey<Ed25519PrivateKey>>,
+
+    #[clap(long)]
+    pub proof_file_path: Option<String>,
+
+    #[clap(long, default_value_t = 2700000000)]
+    pub epk_expiry_date_secs: u64,
+
+    #[clap(long)]
+    pub keyless_jwt: Option<String>,
```

Comment:
let's put all these params inside AccountType::Keyless, so they cannot be set if keyless is not used, and need to be set if keyless is used.

Reply:
talked offline, grouped the args into struct as there's no way to do this for enum without subcommand it seems...